hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -275,6 +276,10 @@ else
   ozonecmd_case "${OZONE_SUBCMD}" "${OZONE_SUBCMD_ARGS[@]}"
 fi
 
+if [[ "${OZONE_SUBCMD_SUPPORTDAEMONIZATION}" == true && "${OZONE_DAEMON_MODE}" =~ ^st(art|op|atus)$ ]] || [[ "${OZONE_SUBCMD}" == classpath ]]; then
+  ozone_validate_classpath
+fi
```

Comment:
Currently we have conditions for validation in two places: here and in `ozone_validate_classpath`.  I'd prefer if all conditions were in the same place (probably best in `ozone_validate_classpath` to simplify the `ozone` executable a bit).

Reply:
Thank you, defined the conditions for validation at a single place!